src/components/signon/SignOn.tsx

```diff
@@ -405,6 +408,24 @@ const SignOn = ({ navigation }: SignOnProps) => {
     showEmptyPasswordError
   ])
 
+  const processReferrerFromClipboard = useCallback(async () => {
+    const hasURL = await Clipboard.hasURL()
+    console.log(`hasURL: ${hasURL}`)
+    if (hasURL !== false) {
+      const contents = await Clipboard.getString()
+      if (contents && contents.indexOf('ref=') > -1) {
+        const handle = contents.match(/ref=([A-z]*)/)[1]
```

Comment:
This regex needs to include any valid `handle` characters, right now it's stripping numbers etc.